<ui version="4.0" >
 <class>NxyterWidget</class>
 <widget class="QWidget" name="NxyterWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>nXYTER</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TabWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>3</number>
     </property>
     <widget class="QWidget" name="BiasTab" >
      <attribute name="title" >
       <string>Bias</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>3</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="BiasGroup" >
         <property name="title" >
          <string>Registers:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="I2CTab" >
      <attribute name="title" >
       <string>I2C</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>3</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="I2CRegsGroup" >
           <property name="title" >
            <string>I2C 32-33 registers</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="OtherRegsGroup" >
           <property name="title" >
            <string>Other registers</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>151</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MasksTab" >
      <attribute name="title" >
       <string>Masks</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>3</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item row="1" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="MaskGroup" >
         <property name="title" >
          <string>Masks:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ThrsholdTab" >
      <attribute name="title" >
       <string>Threshold</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>3</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="ThresholdsGroup" >
         <property name="title" >
          <string>Thresholds</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
